src/org/zaproxy/zap/extension/requester/ManualHttpRequestEditorPanel.java

```diff
@@ -0,0 +1,583 @@
+/*
```

Comment:
We shouldnt use the Paros headers for new files. I think its fine using ZAP ones but stating that it was based on the relevant Paros file